include/nbl/asset/utils/CJITIncludeLoader.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _NBL_ASSET_C_JIT_INCLUDE_LOADER_H_INCLUDED_
+#define _NBL_ASSET_C_JIT_INCLUDE_LOADER_H_INCLUDED_
+
+#include "nbl/asset/utils/IShaderCompiler.h"
+#include "nbl/video/SPhysicalDeviceFeatures.h"
+#include "nbl/video/SPhysicalDeviceLimits.h"
+
+#include <string>
+
+using namespace nbl::asset;
+
+
+namespace nbl::video
+{
+
+class NBL_API2 CJITIncludeLoader : public IShaderCompiler::IIncludeLoader
+{
+public:
+    CJITIncludeLoader();
```

Comment:
it would seem that the constructructor needs same arguments as `collectDeviceCaps` eh?